scripts/cutover_to_pg.sh

```
@@ -0,0 +1,135 @@
+  --nsg-name PrivateProject-nsg \
+  --name TempCutoverAccess \
+  --priority 320 \
+  --direction Inbound \
+  --access Allow \
+  --protocol TCP \
+  --destination-port-ranges 5432 \
+  --source-address-prefixes "$LOCAL_IP" \
+  --output none 2>/dev/null
+
+# Temporarily allow local IP in pg_hba.conf
+ssh "$VM2_HOST" "sed -i '/# Reject everything else/i host    all       monkknows  ${LOCAL_IP}/32 md5' /opt/monkknows-db/pg_hba.conf && cd /opt/monkknows-db && sg docker -c 'docker compose restart db'" 2>/dev/null
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)